src/main-root.css

```diff
@@ -1,13 +1,38 @@
 /**
- * 1. Set the font family of the entire document to use 'system-ui'.
- * 2. Set the font size of the entire document using the default user-agent size
+ * Set a more readable tab size.
+ *
+ * ✌︎ Credit https://github.com/sindresorhus/modern-normalize/issues/17
+ */
+
+:root {
+	tab-size: 4;
+}
+
+/**
+ * 1. Set the font family of the entire document to use a system font.
+ *    ✌︎ Resources and credit
+ *    https://github.com/sindresorhus/modern-normalize/issues/3
+ *    https://github.com/primer/primer/pull/408
+ *    https://infinnie.github.io/blog/2017/systemui.html
+ *    http://markdotto.com/2018/02/07/github-system-fonts/
+ * 2. Set OpenType features for the entire document. Enable kerning,
+ *    common/standard ligatures, contextual ligatures, and proportional,
+ *    oldstyle numerals.
+ *    NOTE: if any of the `font-feature-settings` values aren't needed on child
+ *    elements, e.g.: `<td>`, then you have to explicitly turn them off. Also,
+ *    see the "OpenType features" section in the README.md.
+ * 3. Set the font size of the entire document using the default user-agent size
  *    which is typically 16px.
- * 3. Increase the line height from normalize.css's small '1.15' value. The
+ * 4. Increase the line height from normalize.css's small '1.15' value. The
  *    larger '1.5' value is recommended by the W3C Web Accessibility Initiative.
  */
 
 html {
-  font-family: system-ui, sans-serif; /* 1 */
-  font-size: 1rem; /* 2 */
-  line-height: 1.5; /* 3 */
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif,'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol'; /* 1 */
+  font-feature-settings: 'kern', 'liga', 'clig', 'calt', 'onum', 'pnum'; /* 2 */
+  font-kerning: normal; /* 2 */
+  font-size: 1rem; /* 3 */
```

Comment:
We used to set the base font size in `em` rather than `rem` in Shell.  Not sure why or if one trumps the other?

https://github.com/campaignmonitor/shell/blob/master/src/_base.scss#L37

Reply:
Yeah, `rem` is fine to use too, just equals the user-agent font-size which is typically `16px`.